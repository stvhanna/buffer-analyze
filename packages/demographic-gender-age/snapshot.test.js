// use story.js files as snapshots
import initStoryshots from '@storybook/addon-storyshots';

jest.mock('@bufferapp/add-report');

initStoryshots({
  suit: 'Snapshots',
});
