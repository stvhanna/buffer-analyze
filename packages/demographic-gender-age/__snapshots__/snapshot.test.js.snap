// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshots AudienceGenderAgeChart should render 1`] = `
<span>
  <div
    style={
      Object {
        "width": "750px",
      }
    }
  >
    <div
      className="sc-jKJlTe hrcOlX"
    >
      <header
        className="sc-eNQAEJ heJaNC"
      >
        <h2
          className="sc-hMqMXs klQmSI"
        >
          <span
            style={
              Object {
                "color": "#323b43",
                "fontFamily": "\\"Roboto\\", sans-serif",
                "fontSize": "1.25rem",
                "fontWeight": 600,
              }
            }
          >
            Audience overview 
          </span>
        </h2>
        <div
          className="sc-iAyFgw ixvcxE"
        />
      </header>
      <div
        className="sc-eHgmQL hZzWuy"
        id="js-dom-to-png-gender-age"
      />
    </div>
  </div>
</span>
`;

exports[`Snapshots AudienceGenderAgeChart should render a "no data" state 1`] = `
<span>
  <div
    style={
      Object {
        "width": "750px",
      }
    }
  >
    <div
      className="sc-jKJlTe hrcOlX"
    >
      <header
        className="sc-eNQAEJ heJaNC"
      >
        <h2
          className="sc-hMqMXs klQmSI"
        >
          <span
            style={
              Object {
                "color": "#323b43",
                "fontFamily": "\\"Roboto\\", sans-serif",
                "fontSize": "1.25rem",
                "fontWeight": 600,
              }
            }
          >
            Audience overview 
          </span>
        </h2>
        <div
          className="sc-iAyFgw ixvcxE"
        />
      </header>
      <div
        className="sc-eHgmQL hZzWuy"
        id="js-dom-to-png-gender-age"
      >
        <div
          className="sc-htpNat klaPdF"
        >
          <div
            className="sc-bxivhb cRnhtv"
          >
            <div
              className="sc-ifAKCX kiQAin"
            />
            <h1
              className="sc-EHOje dWbWPz"
            >
              <span
                style={
                  Object {
                    "color": "#59626a",
                    "fontFamily": "\\"Roboto\\", sans-serif",
                    "fontSize": "1rem",
                    "fontWeight": 400,
                  }
                }
              >
                There are no analytics for this date range
              </span>
            </h1>
          </div>
        </div>
      </div>
    </div>
  </div>
</span>
`;

exports[`Snapshots AudienceGenderAgeChart should render a loading state 1`] = `
<span>
  <div
    style={
      Object {
        "width": "750px",
      }
    }
  >
    <div
      className="sc-jKJlTe hrcOlX"
    >
      <header
        className="sc-eNQAEJ heJaNC"
      >
        <h2
          className="sc-hMqMXs klQmSI"
        >
          <span
            style={
              Object {
                "color": "#323b43",
                "fontFamily": "\\"Roboto\\", sans-serif",
                "fontSize": "1.25rem",
                "fontWeight": 600,
              }
            }
          >
            Audience overview 
          </span>
        </h2>
        <div
          className="sc-iAyFgw ixvcxE"
        />
      </header>
      <div
        className="sc-eHgmQL hZzWuy"
        id="js-dom-to-png-gender-age"
      >
        <div
          className="sc-bwzfXH dXsGsm"
        >
          <div
            className="sc-bdVaJa fsNrov"
          >
            <div
              style={
                Object {
                  "textAlign": "center",
                }
              }
            >
              <div
                style={
                  Object {
                    "height": "1rem",
                    "margin": "0 auto 1rem auto",
                    "position": "relative",
                    "width": "1rem",
                  }
                }
              >
                <span>
                  <div
                    style={
                      Object {
                        "height": "1rem",
                        "left": 0,
                        "opacity": 0,
                        "position": "absolute",
                        "top": 0,
                        "transition": "all 1000ms ease-in-out",
                        "width": "1rem",
                      }
                    }
                  >
                    <svg
                      height="100%"
                      style={
                        Object {
                          "fill": "#59626a",
                          "height": "1rem",
                          "width": "1rem",
                        }
                      }
                      version="1.1"
                      viewBox="0 0 16 16"
                      width="100%"
                      xmlns="http://www.w3.org/2000/svg"
                      xmlnsXlink="http://www.w3.org/1999/xlink"
                    >
                      <g
                        className="shuttleGray"
                      >
                        <path
                          d="M1.10241622,4.34038738 L7.75166358,7.40469806 C7.88821853,7.46764349 8.11178147,7.46764349 8.24833642,7.40469806 L14.8975838,4.34038738 C15.0341387,4.27744195 15.0341387,4.17446518 14.8975838,4.11151975 L8.24833642,1.04720908 C8.11178147,0.984263641 7.88821853,0.984263641 7.75166358,1.04720908 L1.10241622,4.11151975 C0.965861261,4.17446518 0.965861261,4.27744195 1.10241622,4.34038738"
                        />
                      </g>
                    </svg>
                  </div>
                </span>
              </div>
              <span
                style={
                  Object {
                    "color": "#59626a",
                    "fontFamily": "\\"Roboto\\", sans-serif",
                    "fontSize": "1rem",
                    "fontWeight": 400,
                  }
                }
              >
                Loading...
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</span>
`;

exports[`Snapshots AudienceGenderAgeChart should render with dropdown for multiple groups 1`] = `
<span>
  <div
    style={
      Object {
        "width": "750px",
      }
    }
  >
    <div
      className="sc-jKJlTe hrcOlX"
    >
      <header
        className="sc-eNQAEJ heJaNC"
      >
        <h2
          className="sc-hMqMXs klQmSI"
        >
          <span
            style={
              Object {
                "color": "#323b43",
                "fontFamily": "\\"Roboto\\", sans-serif",
                "fontSize": "1.25rem",
                "fontWeight": 600,
              }
            }
          >
            Audience overview 
          </span>
        </h2>
        <div
          className="sc-iAyFgw ixvcxE"
        >
          <div
            className="dropdown dropdownContainer"
            style={
              Object {
                "marginLeft": "",
                "marginRight": "0.5rem",
              }
            }
          >
            <a
              className="dropdown__trigger dropdownTrigger"
              onClick={[Function]}
              style={
                Object {
                  "display": "flex",
                  "height": "24px",
                }
              }
            >
              <span
                style={
                  Object {
                    "color": "#323b43",
                    "fontFamily": "\\"Roboto\\", sans-serif",
                    "fontSize": "0.75rem",
                    "fontWeight": 500,
                  }
                }
              >
                Page fans
              </span>
              <span
                style={
                  Object {
                    "marginLeft": "auto",
                    "pointerEvents": "none",
                  }
                }
              >
                <svg
                  height="100%"
                  style={
                    Object {
                      "fill": "#59626a",
                      "height": "0.75rem",
                      "width": "0.75rem",
                    }
                  }
                  version="1.1"
                  viewBox="0 0 16 16"
                  width="100%"
                  xmlns="http://www.w3.org/2000/svg"
                  xmlnsXlink="http://www.w3.org/1999/xlink"
                >
                  <g
                    className="shuttleGray"
                  >
                    <path
                      d="M4.90573685,4.61438297 C4.5159485,4.22537555 4.51510793,3.59709045 4.90996552,3.20500565 L4.90996552,3.20500565 C5.30208929,2.81563546 5.93536183,2.81786315 6.32490003,3.21046581 L11.2750374,8.19954548 C11.6643604,8.59193123 11.6645756,9.22591178 11.2750374,9.61606213 L6.32490003,14.5739785 C5.93557705,14.9639132 5.3048231,14.9648641 4.90996552,14.5699959 L4.90996552,14.5699959 C4.51784174,14.1778616 4.51273549,13.5497061 4.9163418,13.1493324 L8.84038203,9.25672113 C9.03897712,9.05971665 9.04484713,8.74520064 8.84038203,8.54114518 L4.90573685,4.61438297 Z"
                      transform="translate(8.089966, 8.890003) scale(-1, 1) rotate(-270.000000) translate(-8.089966, -8.890003) "
                    />
                  </g>
                </svg>
              </span>
            </a>
            <div
              className="dropdown__content dropdownContent"
            >
              <ul
                className="dropdownList"
              >
                <li
                  className="dropdownListItem"
                >
                  <button
                    aria-label={null}
                    disabled={undefined}
                    onBlur={[Function]}
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseEnter={[Function]}
                    onMouseLeave={[Function]}
                    style={
                      Object {
                        "background": "none",
                        "backgroundColor": "unset",
                        "border": "none",
                        "borderRadius": "unset",
                        "color": "unset",
                        "cursor": "pointer",
                        "display": "unset",
                        "fontFamily": "unset",
                        "fontSize": "unset",
                        "fontWeight": "unset",
                        "lineHeight": "unset",
                        "margin": "unset",
                        "outline": "none",
                        "padding": 0,
                        "transition": "unset",
                      }
                    }
                  >
                    <span
                      style={
                        Object {
                          "alignItems": "center",
                          "boxSizing": "border-box",
                          "display": "flex",
                          "padding": "5px 10px",
                          "position": "relative",
                          "width": "15.84rem",
                        }
                      }
                    >
                      <span
                        style={
                          Object {
                            "color": "#59626a",
                            "fontFamily": "\\"Roboto\\", sans-serif",
                            "fontSize": "0.75rem",
                            "fontWeight": 400,
                          }
                        }
                      >
                        Page fans
                      </span>
                      <div
                        style={
                          Object {
                            "marginLeft": "auto",
                          }
                        }
                      >
                        <svg
                          height="100%"
                          style={
                            Object {
                              "fill": "#168eea",
                              "height": "1rem",
                              "width": "1rem",
                            }
                          }
                          version="1.1"
                          viewBox="0 0 16 16"
                          width="100%"
                          xmlns="http://www.w3.org/2000/svg"
                          xmlnsXlink="http://www.w3.org/1999/xlink"
                        >
                          <g
                            className="curiousBlue"
                          >
                            <path
                              d="M5,12 C5,11.4477153 5.44335318,11 6.0093689,11 L9,11 L9,2.99754465 C9,2.44661595 9.44386482,2 10,2 L10,2 C10.5522847,2 11,2.44483697 11,2.9955775 L11,12.0044225 C11,12.5542648 10.5490248,13 10.009222,13 L5.99077797,13 C5.44358641,13 5,12.5561352 5,12 L5,12 Z"
                              transform="translate(8.000000, 7.500000) rotate(-315.000000) translate(-8.000000, -7.500000) "
                            />
                          </g>
                        </svg>
                      </div>
                    </span>
                  </button>
                </li>
                <li
                  className="dropdownListItem"
                >
                  <button
                    aria-label={null}
                    disabled={undefined}
                    onBlur={[Function]}
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseEnter={[Function]}
                    onMouseLeave={[Function]}
                    style={
                      Object {
                        "background": "none",
                        "backgroundColor": "unset",
                        "border": "none",
                        "borderRadius": "unset",
                        "color": "unset",
                        "cursor": "pointer",
                        "display": "unset",
                        "fontFamily": "unset",
                        "fontSize": "unset",
                        "fontWeight": "unset",
                        "lineHeight": "unset",
                        "margin": "unset",
                        "outline": "none",
                        "padding": 0,
                        "transition": "unset",
                      }
                    }
                  >
                    <span
                      style={
                        Object {
                          "alignItems": "center",
                          "boxSizing": "border-box",
                          "display": "flex",
                          "padding": "5px 10px",
                          "position": "relative",
                          "width": "15.84rem",
                        }
                      }
                    >
                      <span
                        style={
                          Object {
                            "color": "#59626a",
                            "fontFamily": "\\"Roboto\\", sans-serif",
                            "fontSize": "0.75rem",
                            "fontWeight": 400,
                          }
                        }
                      >
                        Post impressions
                      </span>
                    </span>
                  </button>
                </li>
              </ul>
            </div>
          </div>
          <div
            className="sc-hSdWYo indwpk"
          />
        </div>
      </header>
      <div
        className="sc-eHgmQL hZzWuy"
        id="js-dom-to-png-gender-age"
      />
    </div>
  </div>
</span>
`;
